src/vit-testing/mainnet-tools/src/cardano_cli/command/mod.rs

```diff
@@ -45,10 +55,13 @@ pub fn write_to_file_or_println(
     Ok(())
 }
 
+/// Errors for Cardano CLI wrapper
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
+    /// On any parsing error
     #[error(transparent)]
     Parsing(#[from] serde_json::Error),
+    /// On any IO related error
     #[error(transparent)]
     Io(#[from] std::io::Error),
 }
```

Comment:
I wonder if this is better as just a `Report`. I assume that for any error, we just want to print something to `stderr` and exit, given this just wraps a CLI. Or do we need to recover from these errors?